tjsendemail.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.0"  type="plugin" group="system" method="upgrade">
+	<name>PLG_SYSTEM_SENDEMAIL_PLUGINNAME</name>
```

Comment:
change as `plg_system_tjsendemail` here


add this in both lang files at top
`PLG_SYSTEM_TJSENDEMAIL="TJ Send Email"`